src/lib/messaging.hpp

```diff
@@ -35,12 +35,13 @@ class group_factory;
 class msg_service;
 class group_metrics;
 
-class service : public consensus_component {
-    consensus_component::params _start_params;
+class service : public Manager {
```

Comment:
Its a general question as to what is the specialization in "msg_service" compared to "service" class. Name "mesg_service" indicates to me its a derived class of "service", but it is "has a relationship".

Reply:
Also another aspect is service.hpp has "mesg_service" class declaration/definition, but "service" class is declared/defined in messaging.hpp?

Reply:
I've moved all the stuff you shouldn't need to see into `src/lib` where we can rename and re-arrange to our hearts content. We can review the internal names, but the external ones are ok?

Reply:
(the reason this happened was when we merged "sds_messaging" to "nuraft_grpc" to get 'nuraft_messaging')